policies/rudderc/src/backends/metadata.rs

```diff
@@ -389,11 +392,12 @@ struct Input {
 struct SelectOne {
     // actually, the id
     name: String,
+    #[serde(rename = "VARIABLE_NAME")]
```

Comment:
```suggestion
```

No need, everything is already uppercased by line `#[serde(rename_all = "UPPERCASE")]` on the struct